src/l500/ac-trigger.cpp

```diff
@@ -220,10 +237,18 @@ namespace ivcam2 {
         //bool_option::set( value );
 
         // Reset the calibration so we can do it all over again
-        if (auto color_sensor = _autocal->_dev.get_color_sensor())
-            color_sensor->reset_calibration();
-        _autocal->_dev.get_depth_sensor().reset_calibration();
-        _autocal->_dev.notify_of_calibration_change( RS2_CALIBRATION_SUCCESSFUL );
+        try
+        {
+            if (auto color_sensor = _autocal->_dev.get_color_sensor())
+                color_sensor->reset_calibration();
+            _autocal->_dev.get_depth_sensor().reset_calibration();
+            _autocal->_dev.notify_of_calibration_change(RS2_CALIBRATION_SUCCESSFUL);
+        }
+        catch (std::exception const & e)
+        {
+            AC_LOG(ERROR, "EXCEPTION caught in access to device: " << e.what());
+            throw wrong_api_call_sequence_exception("Cannot trigger calibration: depth sensor is not on!");
```

Comment:
We're doing a reset here, not triggering calibration.
If we're going to throw anyway, why are we catching? Why not let the error through?

Reply:
Correct, removed.